CI/PHP-CS-Fixer/code-format.php_cs

```diff
@@ -0,0 +1,26 @@
+<?php
+
+require_once __DIR__ . '/../../libs/composer/vendor/autoload.php';
+
+$finder = PhpCsFixer\Finder::create()
+	->exclude(__DIR__ . '/../../setup/sql')
+	->in(array(
+		__DIR__ .  '/../../cron',
+		__DIR__ .  '/../../include',
+		__DIR__ .  '/../../Modules',
+		__DIR__ .  '/../../Services',
+		__DIR__ .  '/../../setup',
+		__DIR__ .  '/../../src',
+		__DIR__ .  '/../../tests'
+	))
+;
+
+return PhpCsFixer\Config::create()
```

Comment:
This looks very similar to the code formatter I've been testing for the GitHooks.
I found some additional rules that may be should applied, so our coding style matches:
```php
return PhpCsFixer\Config::create()
	->setRules([
		'@PSR2' => true,
		'strict_param' => false,
		'cast_spaces' => true,
		'concat_space' => ['spacing' => 'one'],
		'binary_operator_spaces' => true,
		'unary_operator_spaces' => true,
		'function_typehint_space' => true,
		'return_type_declaration' => ['space_before' => 'one'],
		'whitespace_after_comma_in_array' => true,
	])
	->setFinder($finder);
```
I tried this with the coding style examples provided by #1838 and it worked very well. Maybe you could verify this or we can check this together :+1:

Reply:
@legionth Thx, I've added the config :+1: